recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    - gvconfig_libdir.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - cairo  # [linux]
+    - expat  # [linux]
```

Comment:
>do you have a few minutes to sit down and play with this?

Sorry, not today.  Maybe the best thing to do is to simply insert a comment into `meta.yaml` next to the `expat` requirement, with a link to this discussion.  Then we can come back to it later.